README.md

```diff
@@ -1,23 +1,28 @@
 # Kubecon-India-2024-sponsorship
+
 ![Image](https://github.com/user-attachments/assets/e1e03137-31c6-4322-bee6-d4d8e7f4be57)
 
-* Hello folks 👋
-* We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
---------
+- Hello folks 👋
+- We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
+
+---
 
 ## Criterias:
+
 1. You have to write a blog on topic "How GrowInCommunity helped you in rise the carreer in tech".
 2. After writting the blog, you have to post it on your self own domain (that can be free domain or purchased one).
 3. Share the blog on socials: Twitter/X and LinkedIn with hashtag #growincommunity #kubeconindia2024
-4. We have a dedicated chaanel om our [Discord Server](https://discord.gg/SXqTuNBm4Z), where yoi can communicate wity the team if you have any doubt or need help.
+4. We have a dedicated channel on our [Discord Server](https://discord.gg/SXqTuNBm4Z), where you can communicate with the team if you have any doubt or need help.
 5. You also have to share the bolg link in our discord server channel to verified by the team.
```

Comment:
bolg - > blog .
to verified - . to be verified ?

Reply:
👍